### DISCUSSION TITLE
Expo app crushing on android device after building apk

### DISCUSSION THREAD

**Discussion by mikias21** [software-mansion/react-native-gesture-handler#2968] [Q&A]
Hi everyone, I'm working on a simple app using expo, everything works fine when I run the app on expo but after generating apk file and trying to run I get an error which I've no idea how to fix it. It has been days trying everything but nothing worked for me. Here is the error I'm getting

06-28 05:05:55.195 31505 31570 E ReactNativeJS: Invariant Violation: TurboModuleRegistry.getEnforcing(...): 'RNCDatePicker' could not be found. Verify that a module by this name is registered in the native binary.Bridgeless mode: false. TurboModule interop: false. Modules loaded: {"NativeModules":["HeadlessJsTaskSupport","PlatformConstants","SourceCode","DeviceInfo","UIManager","DeviceEventManager","RNCSafeAreaContext","NativeAnimatedModule","SoundManager","I18nManager","Timing","ReanimatedModule","ImageLoader","RNGestureHandlerModule","StatusBarManager","StatusBarManager"],"TurboModules":[],"NotFound":["NativePerformanceCxx","NativePerformanceObserverCxx","RedBox","BugReporting","NativeReactNativeFeatureFlagsCxx","RNSModule","FrameRateLogger","KeyboardObserver","ModalManager","RNCDatePicker"]}, js engine: hermes
06-28 05:05:55.198 31505 31570 I ReactNativeJS: Running "main
06-28 05:05:55.198 31505 31570 E ReactNativeJS: Invariant Violation: "main" has not been registered. This can happen if:
06-28 05:05:55.198 31505 31570 E ReactNativeJS: * Metro (the local dev server) is run from the wrong folder. Check if Metro is running, stop it and restart it in the current project.
06-28 05:05:55.198 31505 31570 E ReactNativeJS: * A module failed to load due to an error and `AppRegistry.registerComponent` wasn't called., js engine: hermes

if someone has faced similar issue before and have solution please give me a hand.

**Comment by m-bert** [comment]
Hi! This looks more like a problem with `RNCDatePicker`. As you can see, `Gesture Handler` is in the list of loaded modules. Maybe you can find some helpful information in [this issue](https://github.com/software-mansion/react-native-gesture-handler/issues/2847#issuecomment-2193105329).
